src/relay/ir/expr_functor.cc

```diff
@@ -383,11 +383,21 @@ class ExprBinder : public ExprMutator {
     }
   }
 
+  Expr VisitExpr_(const CallNode* op) final {
```

Comment:
We shouldn't need these changes we don't really want to do arbitrary expression to expression replacement.